omniledger/contracts/service.go

```diff
@@ -0,0 +1,27 @@
+package contracts
+
+import (
+	"github.com/dedis/cothority/omniledger/service"
+	"github.com/dedis/onet"
+	"github.com/dedis/onet/log"
+)
+
+func init() {
+	_, err := onet.RegisterNewService("contracts", newService)
```

Comment:
it'd be nice to have a comment here to say this service is only needed to register the contracts but not an actual service